README.md

```diff
@@ -99,15 +106,14 @@ Then run the tests with `go test`
 License
 -------
 
-This is free software under the terms of MIT license (check COPYING file
-included in this package).
+This is free software under the terms of MIT license (check COPYING file included in this package).
 
 Contact and support
 -------------------
 
 The project website is at:
 
-- https://github.com/ncw/swift
+- https://github.com/ncw/swift/v2
```

Comment:
This URL doesn't want to change

Reply:
RIght I was too furios with `sed`ing  :)